src/metrics/api_v2_views.py

```diff
@@ -132,12 +133,19 @@ def ping(request):
 #
 
 @api_view(['GET'])
-def summary(request):
+def summary(request, id=None):
     "returns the final totals for all articles with no finer grained information"
     try:
         kwargs = request_args(request)
         qobj = models.Article.objects.all()
+        if id:
```

Comment:
Built-in `id()` shadowing.

Reply:
if linters like pylint, that are so strict about the smallest things, doesn't care, then I'm going to overlook certain cases as well. shadowing `format` was a bigger deal because it meant you misread the code and it's a function that people actually use. I don't think I've ever ever ever used the built-in `id` function.

Reading it's doc now I can see I've used it's counterpart in other languages often.

If you can integrate a built-in shadowing checker to the lint script, I'll update all the code.

Reply:
👍